sdk/eventgrid/azure-eventgrid/azure/eventgrid/_models.py

```diff
@@ -159,6 +159,17 @@ def model(self):
 
         :rtype: Union[CloudEvent, EventGridEvent]
         """
+        try:
+            event = self._event_dict['type'].rsplit('.')[-1]
```

Comment:
is one of these for cloud_event vs egevent? let's leave helpful comments.

Reply:
removed this - technically it's event_type for EGevent anf type for cloud event :(

Reply:
To be precise, was referring to the variable name (since it contains the type)